src/main/java/org/truffleruby/language/arguments/CheckKeywordArityNode.java

```diff
@@ -32,6 +32,8 @@
 /** Check that no extra keyword arguments are given, when there is no **kwrest */
 public final class CheckKeywordArityNode extends RubyBaseNode {
 
+    public static final RubySymbol[] EMPTY_KEYWORD_ARGUMENTS = new RubySymbol[0];
```

Comment:
When the array component type is part of truffleruby sources, it seems nicer to move the declaration there, in order to avoid multiple constants & instances for empty arrays of that type.
Could you add `RubySymbol.EMPTY_ARRAY` instead?